testing/src/main/groovy/io/opentelemetry/auto/test/utils/ConfigUtils.groovy

```diff
@@ -17,10 +17,9 @@
 package io.opentelemetry.auto.test.utils
 
 import io.opentelemetry.auto.config.Config
-import lombok.SneakyThrows
-
 import java.lang.reflect.Modifier
 import java.util.concurrent.Callable
+import lombok.SneakyThrows
```

Comment:
Can you check your Intellij import settings for groovy?

> With java use the following import layout (groovy should still use the default)

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/CONTRIBUTING.md#intellij-idea

Reply:
What do you mean? Please don't tell me you want to have `java.*` import separately for groovy

Reply:
I'm just following the instructions from `CONTRIBUTING.md`:

![image](https://user-images.githubusercontent.com/218610/87750222-b64e2400-c7af-11ea-8f8f-851190dab727.png)

Reply:
I don't think anyone would objection if you want to change the groovy import ordering to match the java import ordering